Code/GraphMol/Fingerprints/Wrap/MHFPWrapper.cpp

```diff
@@ -151,12 +153,14 @@ BOOST_PYTHON_MODULE(rdMHFPFingerprint) {
   python::class_<MHFPEncoder>(
       "MHFPEncoder",
       python::init<python::optional<unsigned int, unsigned int>>())
-      .def("FromStringArray", &FromStringArray,
-           ((python::arg("vec")),
-            "Creates a MHFP vector from a list of arbitrary strings."))
-      .def("FromArray", &FromArray,
-           ((python::arg("vec")),
-            "Creates a MHFP vector from a list of unsigned integers."))
+      .def("FromStringArray", FromStringArray,
+           FromStringArrayOverloads(
+               (python::arg("vec")),
+               "Creates a MHFP vector from a list of arbitrary strings."))
+      .def("FromArray", FromArray,
+           FromArrayOverloads(
+               (python::arg("vec")),
+               "Creates a MHFP vector from a list of unsigned integers."))
```

Comment:
I think the problem is that the "self" argument is missing and this can just be (not tested):
```suggestion
      .def("FromStringArray", &FromStringArray,
              (python::arg("self"),python::arg("vec")),
               "Creates a MHFP vector from a list of arbitrary strings.")
      .def("FromArray", &FromArray,
               (python::arg("self"),python::arg("vec")),
               "Creates a MHFP vector from a list of unsigned integers.")
```

Reply:
Also, the ampersand is unnecessary since that's not a member function, but rather a helper function defined in the same compilation unit.

Reply:
@greglandrum I have made the change you suggested.